src/desktop/apps/partnerships/templates/meta.jade

```diff
@@ -1,16 +1,4 @@
-if path.indexOf('gallery') > 0
-  title Gallery Partnerships | Artsy
-  meta( property="og:title", content="Gallery Partnerships | Artsy" )
-  meta( name="description", content="Artsy invites the best galleries in the world to list their programs and contribute to the most comprehensive online resource for seasoned and aspiring collectors alike." )
-  meta( name="og:description", content="Artsy invites the best galleries in the world to list their programs and contribute to the most comprehensive online resource for seasoned and aspiring collectors alike." )
-  meta( property="twitter:description", content="Artsy invites the best galleries in the world to list their programs and contribute to the most comprehensive online resource for seasoned and aspiring collectors alike." )
-
-  link( rel="canonical", href=(sd.APP_URL + "/gallery-partnerships") )
-  meta( property="og:url", content=(sd.APP_URL + "/gallery-partnerships") )
-  meta( property="og:image", content=asset('/images/og_image.jpg') )
-  meta( property="og:type", content="website" )
-  meta( property="twitter:card", content="summary" )
-else if path.indexOf('institution') > 0
+if path.indexOf('institution') > 0
```

Comment:
Again - seems like it's only called for institutions now. Maybe we don't even need the conditional checks now?

Reply:
Hm if I'm reading this right, aren't we still using it for Institutions or for auctions? Or is there somewhere else that's overwriting the `Auctions` meta?

Reply:
Oh... have not noticed the last else below... Yep, this one makes sense.